code/backend/pages/03_Delete_Data.py

```diff
@@ -73,6 +73,7 @@ def load_css(file_path):
                 )
                 if len(files_to_delete) > 0:
                     st.success("Deleted files: " + str(files_to_delete))
+                    st.experimental_rerun()
```

Comment:
Looks like this is deprecated in favour of `rerun`. https://docs.streamlit.io/develop/api-reference/execution-flow/st.experimental_rerun

Reply:
@adamdougal  I have made the changes. Can you please review it once.